src/vs/workbench/services/search/common/search.ts

```diff
@@ -50,6 +50,7 @@ export interface ISearchResultProvider {
 
 export interface IFolderQuery<U extends UriComponents = URI> {
 	folder: U;
+	folderName?: string;
```

Comment:
I think either folderName should be non-optional, or it should always be searched when included and `includeFolderName` removed. Can't decide which.

Reply:
Removed `includeFolderName`. The alternative was too messy.